x-pack/plugins/files/server/routes/file_kind/upload.ts

```diff
@@ -69,6 +71,7 @@ export function register(fileKindRouter: FileKindRouter, fileKind: FileKind) {
             output: 'stream',
             parse: false,
             accepts: fileKind.allowedMimeTypes ?? 'application/octet-stream',
+            maxBytes: fileKind.maxSizeBytes ?? fourMiB,
```

Comment:
I only ran into this bug when I tried to upload files from the browser. It turns out there is a min body size which does not necessarily makes sense for file uploads.

I thought we could default to single chunk sized uploads (4 MiB). This might not make sense as a default, happy to go with another value.

An alternative we can consider here is to force plugins to choose max file size (i.e., likely raising the default max body size ~1MB).